api/server.js

```diff
@@ -2,12 +2,46 @@ const express = require("express");
 const app = express();
 const port = process.env.PORT || 4000;
 
-app.use(express.json())
+app.use(express.json());
 
-app.get("/api", (req, res) => 
-{
-  res.status(200)
-	.json({ success: true, msg: "I'm alive!!!" });
-})
+const jobs = require("./exampleJobs.json");
+const rota = require("./exampleRota.json");
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
\ No newline at end of file
+app.get("/api", (req, res) => {
+  res.status(200).json({ success: true, msg: "I'm alive!!!" });
+});
+
+app.get("/jobs", (req, res) => {
+  res.status(200).json(jobs);
+});
+app.get("/rota", (req, res) => {
+  res.status(200).json(rota);
+});
+
+app.post("/rota", (req, res) => {
+  const { title, startDate, endDate, openSpots } = req.body;
+  const newRota = {
+    class: title,
+    startDate,
+    endDate,
+    openSpots,
+  };
+
+  if (
```

Comment:
Nice one @TimeaReich, as we mentioned, we should do the form validation on the frontend. However, having it twice isn't a bad thing :)